corehq/apps/domain/templates/domain/admin/edit_alert.html

```diff
@@ -0,0 +1,12 @@
+{% extends "hqwebapp/bootstrap3/base_section.html" %}
+{% load crispy_forms_tags %}
+{% load i18n %}
+
+{% block page_content %}
+  <form class="form form-horizontal" method="post">
```

Comment:
Doens't crispy forms render the <form> tag already?

Reply:
It does! But if you wish to overwrite anything we add our own and then crispy does not.
Here we want to add the legend node.

Reply:
Oh! So it doesn't add a form insidee the existing form?

Reply:
Nope